stories/internal/ComputeRatio.tsx

```diff
@@ -0,0 +1,192 @@
+// (C) 2019 GoodData Corporation
+import { storiesOf } from "@storybook/react";
+import { screenshotWrap } from "@gooddata/test-storybook";
+import { createHighChartResolver, ScreenshotReadyWrapper } from "../utils/ScreenshotReadyWrapper";
+import * as React from "react";
+import { AreaChart, BarChart, ColumnChart, ComboChart } from "../../src";
+import { ATTRIBUTE_1 } from "../data/componentProps";
+import { measure } from "../../src/helpers/model";
+
+const wrapperStyle = { width: 800, height: 400 };
+
+// create new measure in every component to make sure it's not changed
+const MEASURE_1_RATIO = () =>
+    measure("/gdc/md/storybook/obj/1")
+        .localIdentifier("m1")
+        .ratio();
+
+const MEASURE_2 = () => measure("/gdc/md/storybook/obj/2").localIdentifier("m2");
+
+const MEASURE_2_RATIO = () => MEASURE_2().ratio();
+
+const MEASURE_3 = () => measure("/gdc/md/storybook/obj/3").localIdentifier("m3");
```

Comment:
Measures should be created here and reuse it. We don't need to create in each tests.

Reply:
Unfortunately, the input measure might be changed in each test by function `disableBucketItemComputeRatio` so when reuse it on other tests, the visualization component can receive wrong measure

Reply:
so keep it as is